src/nvim/normal_cmds.lua

```diff
@@ -0,0 +1,1157 @@
+local bit = require('bit')
+
+local NONE = 0
+local NCH = 0x01
+local NCH_NOP = bit.bor(0x02, NCH)
+local NCH_ALW = bit.bor(0x04, NCH)
+local LANG = 0x08
+local SS = 0x10
+local SSS = 0x20
+local STS = 0x40
+local RL = 0x80
+local KEEPREG = 0x100
+local NCW = 0x200
+
+local SEARCH_REV = 0x01
+local BL_WHITE = 1
+local BL_FIX = 4
+
+local FORWARD = 1
+local BACKWARD = -1
+
+local TRUE = 1
+
+local MSCR_DOWN = 0
+local MSCR_UP = 1
+local MSCR_LEFT = -1
+local MSCR_RIGHT = -2
+
+local module = {}
+
+module.nv_cmds = {
+  {
```

Comment:
I assume you copied `ex_cmds.lua`. That file was written before we settled on the `M` convention.
```suggestion
local M = {}

M.nv_cmds = {
  {
```